libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Consensus.hs

```diff
@@ -60,7 +60,7 @@ import qualified Cardano.Ledger.Shelley.PParams as ShelleyPP
 import Cardano.Ledger.Shelley.Rules
 import Cardano.Ledger.Shelley.UTxO (makeWitnessesVKey)
 import Cardano.Ledger.ShelleyMA (ShelleyMAEra)
-import Cardano.Ledger.ShelleyMA.AuxiliaryData (MAAuxiliaryData (..))
+import Cardano.Ledger.ShelleyMA.AuxiliaryData (AllegraTxAuxData (..))
```

Comment:
Once you rebase on master this should dissapear, since it has already been fixed on master

Reply:
that was true